modules/installation-complete-user-infra.adoc

```diff
@@ -146,19 +146,22 @@ command.
 If the pod logs display, the Kubernetes API server can communicate with the
 cluster machines.
 
-ifndef::ibm-power[]
-. For an installation with Fibre Channel Protocol (FCP), additional steps are required to enable multipathing. Do not enable multipathing during installation.
+ifndef::ibm-power,ibm-z[]
+. For an installation with Fibre Channel Protocol (FCP), additional steps are required to enable multipathing.
 +
 [NOTE]
 ====
 When installing with multipath, it is strongly recommended to enable it at installation time, and not at a later time, which can cause problems.
 ====
 +
 See "Enabling multipathing with kernel arguments on {op-system}" in the _Installing on bare metal_ documentation for more information.
-endif::ibm-power[]
+endif::ibm-power,ibm-z[]
 ifdef::ibm-power[]
+// Should the following sentence start with "For an installation with Fibre Channel Protocol (FCP)" as it does in the ibm-z ifdef?
```

Comment:
@ktania46 or @SNiemann15 - Should the IBM Power ifdef also preface with FCP as we do in IBM Z on line 201?

Reply:
FCP is not part of Power, so that's why it had been excluded previously